app/1.0/blog/2016-10-03-1.7-release.md

```diff
@@ -0,0 +1,25 @@
+---
+title:  "1.7 Release"
+
+---
+_We announce significant releases here on the blog, highlighting the major new features as well as breaking changes. As always, we'd love to get your feedback, issues, and PR's, so tweet us [@polymer](https://twitter.com/polymer) or contribute to the [Github](https://github.com/Polymer)._
+
+We're pleased to announce the release of:
+
+## Polymer 1.7
+
+The primary goal of the upcoming Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components. With this in mind, Polymer 1.7.0 provides forwards compatibility and initial support for migration. 
+
+### `<slot>` element 
+
+Elements can opt-in to using the new `<slot>` element and `::slotted()` CSS selector from ShadowDOM v1.
```

Comment:
Also maybe add a second sentence: These replace the `<content>` element and `::content` selector in shadow DOM v0. The v1 API is somewhat more restrictive, so you should read up on the details before making changes. 

_Cross-reference to come_